client/src/components/OverlayComponent/OverlayComponent.scss

```diff
@@ -0,0 +1,25 @@
+@import '@/variables';
+
+.modal-overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  height: 100vh;
+  background-color: rgba(0, 0, 0, 0.5);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  z-index: 1000;
+}
+
+.modal-content {
+  background-color: $background;
+  padding: 20px;
+  border-radius: 8px;
+  box-shadow: 0 4px 10px rgba(0, 0, 0, 0.1);
+  min-width: 300px;
+  max-width: 500px;
+  text-align: center;
+  border: 2px solid $component-border-color;
+}
```

Comment:
I am pretty sure we have a modal component in the repo, but I am open to a new one if it is more fitting